### PR TITLE
tmh css

### PR DESCRIPTION
**Pull request by Polkas**
linked to https://github.com/insightsengineering/coredev-tasks/issues/225

- div + display="inline" == span
- remove br() tag as <br /> would have different result in different browsers.
- consistent loading of css with tagList(include_css, main_tag)
- add style at the head not body

### CODE REVIEW

**Review by gogonzo — APPROVED** [review]
LGTM

File: R/pca.R [hunk 1]
```diff
@@ -67,82 +67,87 @@ ui_g_pca <- function(id,
...
-            DT::DTOutput(ns("table_pca"))
-          )
-        ),
-        tabPanel(
-          "PC and Sample Correlation",
-          column(
-            width = 12,
-            div(style = "height:20px;"),
-            plotOutput(ns("plot_cor")),
-            br(), br(), br(),
```
Comment (nikolas-burkoff) [inline R/pca.R:138]: these not needed any more

File: R/geneSpec.R [hunk 2]
```diff
@@ -109,8 +109,8 @@ geneSpecInput <- function(inputId,
         options = list(
           render = I("{
           option: function(item, escape) {
-              return '<div> <div style = \"font-size: inherit; display: inline\">' + item.label + '</div>' +
-                ' <div style=\"color: #808080; font-size: xx-small; display: inline\" >' + item.value + '</div> </div>'
+              return '<div> <span style = \"font-size: inherit;\">' + item.label + '</div>' +
+                ' <span style=\"color: #808080; font-size: xx-small;\" >' + item.value + '</div> </div>'
```
Comment (nikolas-burkoff) [inline R/geneSpec.R:113]:
```suggestion
                ' <span style = \"color: #808080; font-size: xx-small;\" >' + item.value + '</div> </div>'
```

Comment (Polkas) [reply R/geneSpec.R:113]: it is html so not need for a space

File: R/pca.R (hunk 1 above)
Comment (Polkas) [reply R/pca.R:138]: what you mean

Comment (nikolas-burkoff) [reply R/pca.R:138]: The `br(), br(), br()` is missing now?
image

Is that OK?

File: R/geneSpec.R [hunk 3]
```diff
@@ -109,8 +109,8 @@ geneSpecInput <- function(inputId,
         options = list(
           render = I("{
           option: function(item, escape) {
-              return '<div> <div style = \"font-size: inherit; display: inline\">' + item.label + '</div>' +
-                ' <div style=\"color: #808080; font-size: xx-small; display: inline\" >' + item.value + '</div> </div>'
+              return '<div> <span style = \"font-size: inherit;\">' + item.label + '</div>' +
```
Comment (Polkas) [inline R/geneSpec.R:112]:
```suggestion
              return '<div> <span style=\"font-size: inherit;\">' + item.label + '</div>' +
```

File: R/pca.R (hunk 1 above)
Comment (Polkas) [reply R/pca.R:138]: my-5 replace the upper div and the lowe br's. It is ok.

**Pull request merged**